RevenueCat/Scripts/Purchases.cs

```diff
@@ -1286,6 +1286,34 @@ public void RedeemWebPurchase(WebPurchaseRedemption webPurchaseRedemption, Redee
         _wrapper.RedeemWebPurchase(webPurchaseRedemption);
     }
 
+    public delegate void GetVirtualCurrenciesFunc(VirtualCurrencies? virtualCurrencies, Error error);
```

Comment:
Here, I've marked `virtualCurrencies` as optional (with `VirtualCurrencies?`). This doesn't seem to be the convention in this file though, other functions seem to omit the `?` in both the return object _and_ the optional error.

Do y'all think it's better to include the `?` here, or stick to the conventions in the API and make this just be a `VirtualCurrencies` type?

Same goes for the `public VirtualCurrencies? GetCachedVirtualCurrencies()` function below

Reply:
That's a great point! So from what I read, nullable reference types have been enabled by default in C# for a while but seems that Unity doesn't have them enabled... So basically adding the `?` would just be a bit of an indicator, but is not enforced even if not added...

I do think it's probably worth to start making the types nullable since it seems more correct. Great catch!!

In this case, we should also make `Error` nullable right?